### ISSUE TITLE
remove brute-force oci-sif vs. native sif determination...?

### ISSUE THREAD

**Issue by preminger** [sylabs/singularity#2184]
It seems like the code we now have in `pkg/image` satisfies the precondition in this TODO:

https://github.com/sylabs/singularity/blob/76ccdddf6323fd11c9444c8251d3474570c93ced/cmd/internal/cli/actions.go#L207C8-L207C8

```golang

	// TODO - drop once we have implemented prefix-less oci-sif vs native sif detection.
	if isOCI {
		image = "oci-sif:" + image
	}

```

Is that correct? And if so, is it time to address this?

**Comment by dtrudg** [comment]
Will need to trace it through... it's not impossible that some other code is still looking at the prefix.

**Comment by preminger** [comment]
Yeah, for sure - and this is complicated by the fact that there's code later in the flow that adds (or adds back, as the case may be) the prefix: https://github.com/preminger/singularity/blob/8a6e709de030a224bc98e5113bbe4da6113e9b63/internal/pkg/runtime/launcher/oci/launcher_linux.go#L897

(Though in this particular case, it's doing so intelligently, based off of the actual contents of the image.)

**Comment by dtrudg** [comment]
:+1: If you'd like to work through it, definitely a fine time to do so.
